examples/contracts/src/lib.rs

```diff
@@ -791,4 +797,33 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    async fn storage_slots_override() -> Result<()> {
+        {
+            // ANCHOR: storage_slots_override
+            use fuels::{programs::contract::Contract, tx::StorageSlot};
+            let slot_override = StorageSlot::new([1; 32].into(), [2; 32].into());
```

Comment:
Minor UX-related suggestion: how hard would it be to change this signature from `[1; 32].into()` to `[1; 32]` (i.e., directly accepting the slice)?

Reply:
We'd need to have our own wrapper around `StorageSlot`. Currently, it is just being exported from `fuel_tx`.

Some care would also be needed to check that we consistently use our wrapper and not mix-match with the `fuel_tx` one.

We might also want to consider at some point if we have any benefit in the SDK from the wrapper around `[u8; 32]` that `Bytes32` is. They're causing this `into` ux snag here.

I'd rate it a 4/10 on a PITA scale. Drop a msg and I'll implement it.

Reply:
I would not add the wrapper

Reply:
I'm pushing this for later; I don't want to stall this work because of this discussion.